README.md

```diff
@@ -1,12 +1,11 @@
-
 [English Docs](README.md)  |  [中文文档](doc/README-zh.md)
 
 
 # HarmonyOS
 ![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg) 
 
 <p align="center">
-  <a href="https://github.com/Awesome-HarmonyOS/HarmonyOS">
+  <a href="https://github.com/leisim/awesome-fuchsia">
```

Comment:
explain?

Reply:
I don't know where this comes from. I just forked and replaces HormonyOS with HarmonyOS. Sorry.

Reply:
I probably forked it very close to when it was fixed by @Awesome-HarmonyOS 
https://github.com/Awesome-HarmonyOS/HarmonyOS/commit/a3ba08a0712b8a4f79dd4dfc23d48992d6cdd568